dmscripts/notify_buyers_when_requirements_close.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime, timedelta
+
+import dmapiclient
+from dmutils.formats import DATE_FORMAT, DATETIME_FORMAT
+from dmutils.email import send_email, MandrillException
+
+from .email import render_html, get_sent_emails
+
+from . import logging
+
+logger = logging.configure_logger({'dmapiclient': logging.INFO})
+
+
+def get_closed_briefs(data_api_client, closed_at):
+    return [
+        brief for brief in data_api_client.find_briefs_iter(status='closed', with_users=True)
+        if datetime.strptime(brief['applicationsClosedAt'], DATETIME_FORMAT).date() == closed_at
```

Comment:
Maybe `closed_on` rather than `closed_at`? Or even `date_closed_on` to make it even clearer.